File: 1043/A/prog.py
import sys


def main(inp, out):
    n = int(inp.readline())
    a = list(map(int, inp.readline().split()))
    s = sum(a)
    answer = max(2 * s // n + 1, max(a))
    print(answer, file=out)


if __name__ == '__main__':
    if sys.platform == 'win32':
        main(sys.stdin, sys.stdout)
    else:
        with open("input.txt", 'r') as inp, open("output.txt", 'w') as out:
            main(inp, out)

File: 1043/A/prog.cpp
#include <bits/stdc++.h>


template <class Class>
void Print(Class instance) {
    std::cout << instance << std::endl;
}

void Print() {
    std::cout << std::endl;
}


int main() {
    std::ios_base::sync_with_stdio(false);
    #ifdef LOCAL
        std::ifstream input("input.txt");
        std::ofstream output("output.txt");
    #else
        auto& input = std::cin;
        auto& output = std::cout;
    #endif
    size_t n;
    input >> n;
    std::vector<size_t> a(n);
    size_t sum = 0;
    size_t max = 0;
    for (size_t index = 0; index < n; ++index) {
        input >> a[index];
        sum += a[index];
        max = std::max(max, a[index]);
    }
    size_t result = std::max(max, 2 * sum / n + 1);
    output << result;
    return 0;
}
